Tasks/DeployTestAgent/TestAgentConfiguration.ps1

```diff
@@ -151,6 +155,7 @@ function Get-TestAgentConfiguration
     Write-Verbose -Message ("Existing Configuration : EnableAutoLogon : {0}" -f $enableAutoLogon) -Verbose
     Write-Verbose -Message ("Existing Configuration : DisableScreenSaver : {0}" -f $disableScreenSaver) -Verbose
     Write-Verbose -Message ("Existing Configuration : RunningAsProcess : {0}" -f $runningAsProcess) -Verbose
+    Write-Verbose -Message ("Existing Configuration : Capabilities : {0}" -f $capabilities) -Verbose
```

Comment:
Capabilities is Null or Empty. If Null or Empty is there in string then we do not pass /Capabilities .